Error when compiling

I compiled the code successfully yesterday, but today when I compile the newest version of code on branch master, I get the following error.
```
-- Using /usr/lib64/ccache/c++ (compiler id: GNU)
CMake Error at /usr/share/cmake3/Modules/FindPackageHandleStandardArgs.cmake:164 (message):
  MySQL_INCLUDE_DIR (missing: MySQL_LIBRARY)
Call Stack (most recent call first):
  /usr/share/cmake3/Modules/FindPackageHandleStandardArgs.cmake:445 (_FPHSA_FAILURE_MESSAGE)
  cmake/FindMySQL.cmake:50 (find_package_handle_standard_args)
  CMakeLists.txt:40 (find_package)

-- Configuring incomplete, errors occurred!
```
I do not understand why it says I am missing MySQL_LIBRARAY, help.

--- a/cmake/FindMySQL.cmake
+++ b/cmake/FindMySQL.cmake
@@ -42,7 +42,10 @@ else()
             NAMES mysql_version.h
             PATH_SUFFIXES mysql
         )
-        find_library(MySQL_LIBRARY NAMES mysqlclient mysqlclient_r)
+        find_library(MySQL_LIBRARY
+            NAMES mysqlclient mysqlclient_r
+            PATH_SUFFIXES mysql  # for CentOS 7
+        )
     endif()
 endif()
 

